src/main/drivers/barometer/barometer_bmp280.c

```diff
@@ -37,7 +37,47 @@
 
 #include "barometer_bmp280.h"
 
-#if defined(USE_BARO) && (defined(USE_BARO_BMP280) || defined(USE_BARO_SPI_BMP280))
+#if defined(USE_BARO_BMP280)
```

Comment:
This may cause problems in some targets with SPI based Baro sensors the definition without the SPI does not exists. This code will remove the Baro driver completely for this targets. In the past is was common practice to only define what is installed. The means having USE_BARO_BMP280 or USE_BARO_SPI_BMP280 individually or combined was and acceptable and working option.